WalletWasabi/Userfacing/PasswordHelper.cs

```diff
@@ -68,7 +69,7 @@ public static bool IsTooLong(int length)
 			return length > MaxPasswordLength;
 		}
 
-		public static bool IsTrimmable(string? password, out string? trimmedPassword)
+		public static bool IsTrimmable(string? password, [NotNullWhen(true)] out string? trimmedPassword)
```

Comment:
Please don't create mixed PRs, this makes it harder to review. Also I think this isn't good. This should be always not null IMO.